src/components/TokenBottomSheet.tsx

```diff
@@ -93,6 +96,8 @@ function TokenBottomSheet<T extends TokenBalance>({
   tokens,
   searchEnabled,
   title,
+  titleStyle,
+  useTokenBalanceItem,
```

Comment:
it's not obvious to me what it actually means to use the token balance item component vs TokenOption. Is TokenBalanceItem preferable for new code? 

I just left a similar [comment](https://github.com/valora-inc/wallet/pull/4322#discussion_r1366180095) on Jean's PR here-- I think we should avoid using boolean parameters whenever we can. In this case, you could write a new version of TokenBottomSheet, and a legacy version that shares most of the code besides the TokenBalanceItem/TokenOption part.